compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1511,4 +1511,30 @@ computeDimUpperBound(Value shapedValue, unsigned dimNum,
   return roundedDimBound.getSize();
 }
 
+static bool isAllConstantValue(ArrayRef<OpFoldResult> ofrs, int64_t v) {
+  return llvm::all_of(
+      ofrs, [&](OpFoldResult ofr) { return isConstantIntValue(ofr, v); });
+}
+
```

Comment:
nit: I recently added this upstream as `areAllConstantIntValue`, so we don't need this function anymore.